CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v1.0.0
```

Comment:
https://github.com/racklet/render-drawio-action/pull/1 reaches `v0.1.0` while this goes straight to `v1.0.0`?

Reply:
It should be v1.0.0 in both places, fixed